test/gtest/Makefile.am

```diff
@@ -208,6 +208,7 @@ noinst_HEADERS = \
 	ucp/ucp_test.h \
 	ucp/ucp_datatype.h
 
+if HAVE_GTEST
```

Comment:
maybe in configure.ac take out this whole makefile out of AC_CONFIG_FILES in case gtest is not enabled?

Reply:
it is more complicated, but should work